### PR TITLE
[RHACS] Updated the support page for uploading diagnostic bundle

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/OSDOCS-2811

- Removed OpenShift specific information
- Added diagnosting bundle generation steps.

Preview: https://openshift-docs-git-osdocs-2811-fix-gnelson.vercel.app/openshift-acs/master/support/getting-support.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@IAmATeaPot418 PTAL

**Comment by IAmATeaPot418** [comment]
Following up with support on this.

**Comment by gaurav-nelson** [comment]
Received an email with "Looks good" from support team (Aaron Strouse).

File: support/getting-support.adoc [hunk 1]
```diff
@@ -27,4 +27,18 @@ provide specific details, such as the section name and {product-title} version.
 
 include::modules/support-knowledgebase-about.adoc[leveloffset=+1]
 include::modules/support-knowledgebase-search.adoc[leveloffset=+1]
+
```
Comment (kalexand-rh) [inline support/getting-support.adoc:30]: You need an anchor here.

File: support/getting-support.adoc [hunk 2]
```diff
@@ -27,4 +27,18 @@ provide specific details, such as the section name and {product-title} version.
...
+== Generating a diagnostic bundle
+You can generate a diagnostic bundle and send that data to enable the support team to provide insights into the status and health of Red Hat Advanced Cluster Security for Kubernetes components.
+
+[NOTE]
+====
+The diagnostic bundle is unencrypted, and typically, the size of the bundle is between 100 KB and 1 MB (depending upon the number of clusters in your environment).
```
Comment (kalexand-rh) [inline support/getting-support.adoc:36]: > Avoid using parentheses in general text because text in parentheses is decreased in importance.

**Review by kalexand-rh — COMMENTED** [review]
I see two small things, then this is good!

**Pull request merged**